### PR TITLE
Added Signal Desktop Beta package version 5.2.0-beta.1

### PR DESCRIPTION
**Pull request by bearmannl**
added signal beta package version  5.2.0-beta.1

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18163)

### CODE REVIEW

**Comment by bearmannl** [comment]
Initially copied the main info over from Signal.Signal. Realized now that the folder & package naming convention wasn't followed by using Publisher.Package since Open Whisper Systems is the publishers, as provided by Open Whisper Systems themselves in the installer file.

**Comment by bearmannl** [comment]
> Initially copied the main info over from Signal.Signal. Realized now that the folder & package naming convention wasn't followed by using Publisher.Package since Open Whisper Systems is the publishers, as provided by Open Whisper Systems themselves in the installer file.

@OfficialEsco, as asked here: https://github.com/microsoft/winget-pkgs/pull/2155#issuecomment-864517403

If there is a valid reason for going with Signal.Signal, I would suggest doing the same for the Beta.

File: manifests/o/OpenWhisperSystems/Signal-Beta/5.2.0-beta.1/OpenWhisperSystems.Signal-Beta.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: OpenWhisperSystems.Signal-Beta
```
Comment (OfficialEsco) [inline manifests/o/OpenWhisperSystems/Signal-Beta/5.2.0-beta.1/OpenWhisperSystems.Signal-Beta.installer.yaml:4]:
```suggestion
PackageIdentifier: OpenWhisperSystems.Signal.Beta
```
Lets do this so it looks cleaner (this is a new feature which is why older manifests use `-`

**Comment by OfficialEsco** [comment]
All your PR's says this
`Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\o\OpenWhisperSyste***\Signal.Beta\5.2.0-beta.1', expected 'manifests\o\OpenWhisperSyste***\Signal\Beta\5.2.0-beta.1'`

**Comment by bearmannl** [comment]
> All your PR's says this
> `Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\o\OpenWhisperSyste***\Signal.Beta\5.2.0-beta.1', expected 'manifests\o\OpenWhisperSyste***\Signal\Beta\5.2.0-beta.1'`
> 
> Nvm this is different, you gotta change the path to `OpenWhisperSyste***\Signal\Beta\5.2.0-beta.1` from `OpenWhisperSyste***\Signal.Beta\5.2.0-beta.1`

Thanks, just finished correcting the other ones. Made a PowerShell booboo.

Working on this one now.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**